### ISSUE TITLE
Additional instructions needed in 12.4.3.1

### ISSUE THREAD

**Issue by maislind** [jennybc/happy-git-with-r#79]
I handed this Book off to a colleague to get started with Git (and save me much time, thank you!). They got to 12.4.3.1 without a problem, but I had to field questions on the additional step of creating an SSH config file for macOS >= 10.12.2. It would be great if there were a reference or steps about this in the Book. For many people, I imagine, this may be their first interaction with anything shell related.

**Comment by maislind** [comment]
I do realize there is a link right below it, which is helpful, although not perfect for the completely uninitiated. Having a single source to point to and avoid having to say "here is GitHub, ssh, and by the way, you may have to jump over to stack which immediately links to over to a GitHub post (another two things you do not, or not yet, use)" would be helpful.

**Comment by jennybc** [comment]
This is currently in the book, in the section we're talking about:

<img width="996" alt="screen shot 2018-03-07 at 9 10 38 am" src="https://user-images.githubusercontent.com/599454/37106768-80d5e4e0-21e7-11e8-9a46-c0d4ec7a414c.png">

So are you saying it's ... not enough?

**Comment by maislind** [comment]
Correct, for my particular use, this is not enough. The people I immediately need to teach are currently comfortable with getting software licenses renewed once per year at the campus bookstore, and when there are issues, having a phone number to call. This group has a different mindset than, for example, Stuff they forget Rstudio Conf 2018 (with a 100% success rate), and they are looking for reasons to complain its hard and not come aboard.

After recent feedback from my handing off the instructions, I re-read the section, and there seems to be an easily fillable hole. 

Without more details in the section, here is the criticism (emphasis added for clarity):  
1.    Where is this file supposed to go? So, I had to go to an external resource and scroll down the page to the first answer and read to the ~20th line? Who wrote this post? I was just using a reference written by a Statistics professor, and now I am changing things on my computer because someone in a forum said to?
2.    Okay, but I cannot see a “.ssh” directory, what and where is that? Also, are we messing with files we should not touch? A quick Google says messing with them can be dangerous, and I did have to Google where to find the folder or ask for external support.
3.    Okay, now I see the folder. How do I add the file? What type of file should I add? 
4.    Okay, now I can see I can add things. In this forum post there is an additional line in the config, should I add only the lines in the Happy Git with R book, or should I add these? 

For some people, this has gone from a smooth process to potentially quite overwhelming. One small additional instruction set would steer 95% of users around thinking about the above points, perhaps something like:

If you have MacOS > xx.x then: 
1.    Go to the Finder and press Command+Shift+G and type “~/.ssh”. 
2.    Add file by xxx and type these lines xxx 
3.    If you need more help, see the link.

**Comment by jennybc** [comment]
Would you like to make a PR?
